drivers/modem/Kconfig.gsm

```diff
@@ -79,6 +79,14 @@ config MODEM_GSM_ATTACH_TIMEOUT
 	  using AT+CGATT. This setting dictates how much time in seconds
 	  we give the modem before giving up.
 
+config MODEM_GSM_REGISTER_TIMEOUT
+	int "Timeout for registering to cellular tower"
+	default 60
```

Comment:
I think this default time should be much larger. On a modem that has never been on the network, this can take minutes. A good default for this would be something like 300 seconds.

Reply:
fixed